plugins/in_opentelemetry/opentelemetry_logs.c

```diff
@@ -368,6 +368,7 @@ static int process_json_payload_log_records_entry(struct flb_opentelemetry *ctx,
     msgpack_object_map *log_records_entry;
     msgpack_object     *timestamp_object;
     uint64_t            timestamp_uint64;
+    char *logs_body_key;
```

Comment:
If the comment from line 1503 is correct and you do make changes to the code please adapt this line so it follows the predominant style in the function. 

You don't need to follow this style in your own code but when you do modify pre-existing code please try to retain the style to keep things tidy and readable, not because one is better than the other but because it's easier to read homogenous code.

Reply:
Ah thanks, I missed that this block of variables was indented (originally had the variable declared elsewhere and moved it up).